docs/modules/ROOT/pages/how-tos/cloudscale/decommission.adoc

```diff
@@ -127,6 +127,12 @@ Go to https://control.vshn.net/syn/lieutenantclusters
 +
 - Delete cluster entry using the delete button
 
+. Delete all volumes in cloudscale.ch project
```

Comment:
```suggestion
. Delete all remaining volumes which were associated with the cluster in the cloudscale.ch project.
+
TIP: This step is required because the csi-cloudscale driver doesn't have time to properly cleanup PVs when the cluster is decommissioned with `terraform destroy`.
```